### DISCUSSION TITLE
No funciona "select" query parameter en `GET /surveys`

### DISCUSSION THREAD

**Discussion by Mauriceac** [Cotalker/documentation#574] [4. Bug Reports]
No funciona "select" query parameter en `GET /surveys`.

REQUEST SAMPLE:
```bash
curl --location --request GET 'https://www.cotalker.com/api/v2/surveys?select=code&debug=true' \
--header 'admin: true' \
--header 'Authorization: Bearer $ACCESS_TOKEN'
```

RESPONSE SAMPLE:
```json
{
    "data": {
        "surveys": []
    },
    "debug": {
        "query": [
            {
                "company": "ObjectId<COTCompany>",
                "select": "code",
                "isActive": true
            }
        ]
    }
}
```

**Comment by acedward** [comment]
Cuando va sin "select" esto funciona?

**Reply by Mauriceac** [reply]
Funciona correctamente sin "select".

**Reply by acedward** [reply]
OK es claramente un bug
En https://doc.cotalker.com/docs/documentation/models/surveys/model_surveys existe el campo `code`

**Reply by acedward** [reply]
Se abrió un issue referenciando esta discussion.
Si hay una necesitad crítica, podemos resolverlo con urgencia. Vamos a dejarlo por mientras en la cola de bugs de prioridad media.

Gracias por el reporte!
Nos ayuda a mejorar el producto
